{ 'high_step_bkp': { 'left_arm': [ { 'duration': 2.952,
                                     'name': u'high_step_5',
                                     'positions': [ -1.6698,
                                                    -0.0332,
                                                    -0.6098,
                                                    0.0544,
                                                    -0.0012,
                                                    0.0001,
                                                    0.0],
                                     'starttime': 0.0}],
                     'left_leg': [ { 'duration': 3.016,
                                     'name': u'high_step_0',
                                     'positions': [ 0.0099,
                                                    -0.0536,
                                                    0.0021,
                                                    -0.0014,
                                                    -0.0063,
                                                    -0.0546],
                                     'starttime': 0.0},
                                   { 'duration': 4.024,
                                     'name': u'high_step_0_left',
                                     'positions': [ -0.0038,
                                                    -0.1993,
                                                    0.0007,
                                                    -0.0005,
                                                    0.0027,
                                                    -0.1973],
                                     'starttime': 3.016},
                                   { 'duration': 2.932,
                                     'name': u'high_step_1_left',
                                     'positions': [ 0.0008,
                                                    -0.1963,
                                                    -0.2525,
                                                    0.5059,
                                                    0.2557,
                                                    -0.2007],
                                     'starttime': 7.04},
                                   { 'duration': 4.192,
                                     'name': u'high_step_2_left',
                                     'positions': [ 0.01,
                                                    -0.3279,
                                                    -1.4502,
                                                    2.4264,
                                                    0.9951,
                                                    -0.22],
                                     'starttime': 9.972},
                                   { 'duration': 2.83,
                                     'name': u'high_step_3_left',
                                     'positions': [ 0.0102,
                                                    -0.3275,
                                                    -1.517,
                                                    1.5889,
                                                    0.1983,
                                                    -0.2201],
                                     'starttime': 14.164},
                                   { 'duration': 2.952,
                                     'name': u'high_step_4_left',
                                     'positions': [ 0.01,
                                                    -0.4275,
                                                    -1.3728,
                                                    1.5538,
                                                    0.2636,
                                                    -0.3378],
                                     'starttime': 16.994},
                                   { 'duration': 3.147,
                                     'name': u'high_step_5_left',
                                     'positions': [ 0.01,
                                                    -0.2576,
                                                    -1.43,
                                                    1.66,
                                                    0.33,
                                                    -0.1878],
                                     'starttime': 19.946}],
                     'neck': [],
                     'right_arm': [ { 'duration': 2.952,
                                      'name': u'high_step_5',
                                      'positions': [ 1.6698,
                                                     0.0332,
                                                     0.6098,
                                                     -0.0544,
                                                     0.0012,
                                                     -0.0001,
                                                     -0.0],
                                      'starttime': 0.0}],
                     'right_leg': [ { 'duration': 3.016,
                                      'name': u'high_step_0',
                                      'positions': [ -0.0099,
                                                     0.0536,
                                                     -0.0021,
                                                     0.0014,
                                                     0.0063,
                                                     0.0546],
                                      'starttime': 0.0},
                                    { 'duration': 4.002,
                                      'name': u'high_step_0_right',
                                      'positions': [ 0.0225,
                                                     -0.1666,
                                                     -0.0149,
                                                     0.0022,
                                                     0.0094,
                                                     -0.1544],
                                      'starttime': 3.016},
                                    { 'duration': 3.145,
                                      'name': u'high_step_5_right',
                                      'positions': [ -0.01,
                                                     0.0662,
                                                     -0.0425,
                                                     0.08,
                                                     -0.0629,
                                                     0.05],
                                      'starttime': 19.995}],
                     'torso': [ { 'duration': 3.03,
                                  'name': u'high_step_0',
                                  'positions': [-0.0, 0.0],
                                  'starttime': 0.0}]}}
